### PR TITLE
Removes Air Conditioner

### PR DESCRIPTION
**Pull request by variableundefined**
## What Does This PR Do

Remove the unused chiller / air conditioner in chiller.dm

Also updates the tutorial and examples for TGUI a bit

## Why It's Good For The Game

1. It has no sprites (See any aircond0?)
image
2. It is unused in any map whatsoever
3. It is a whole load of code to do the same thing Space Heater can do

## Changelog
:cl:
del: Removed unused air conditioner from code.
/:cl:

### CODE REVIEW

File: tgui/docs/tutorial-and-examples.md [hunk 1]
```diff
@@ -294,7 +294,7 @@ here's what you need (note that you'll probably be forced to clean your shit up
 upon code review):
 
 ```dm
-/obj/copypasta/ui_interact(mob/user, ui_key = "main", datum/tgui/ui = null, force_open = 0, datum/tgui/master_ui = null, datum/ui_state/state = default_state) // Remember to use the appropriate state.
+/obj/copypasta/ui_interact(mob/user, ui_key = "main", datum/tgui/ui = null, force_open = FALSE, datum/tgui/master_ui = null, datum/ui_state/state = default_state) // Remember to use the appropriate state.
```
Comment (farie82) [inline tgui/docs/tutorial-and-examples.md:297]: bit odd to include it in this PR

**Review by farie82 — APPROVED** [review]

File: tgui/docs/tutorial-and-examples.md (hunk 1 above)
Comment (AffectedArc07) [inline tgui/docs/tutorial-and-examples.md:297]:
```suggestion
/obj/copypasta/ui_interact(mob/user, ui_key = "main", datum/tgui/ui = null, force_open = FALSE, datum/tgui/master_ui = null, datum/ui_state/state = GLOB.default_state) // Remember to use the appropriate state.
```
If youre doing this in the PR, do this too

**Review by AffectedArc07 — APPROVED** [review]
Yeah purge these

**Comment by FreeStylaLT** [comment]
Remove, don't remove

**Comment by MRgaty** [comment]
Improve, dont prove.

**Review by Dumbdumn5 — APPROVED** [review]
Though they look cool, I've never seen these and I don't imagine they'll be missed if they're a hassle to upkeep. :+1:

**Review by AffectedArc07 — APPROVED** [review]

**Comment by Spacemanspark** [comment]
Man these were such a cool addition to the server, why'd it have to get iced

:^)

**Pull request merged**